### PR TITLE
add basics terminology section (link to demo in comment)

### PR DESCRIPTION
**Pull request by cluelessperson**
https://cluelessperson.com/share/public/electrum/docs/basics.html

Describes basic "must know" terminology.

### CODE REVIEW

File: basics.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Transaction Fee
+---------------
+
+This fee pays for the amount of data in the transaction.  Too low and the transaction will not confirm.
+Transactions with the highest fees usually go first.  After a long time, unconfirmed transactions will drop, as if it never happened.
+
+Confirmation
+------------
+
+Confirmations occur when a transaction becomes permanently stored in the blockchain.  On average, confirmations will occur within 10 minutes.
```
Comment (bauerj) [inline basics.rst]: That's not true. It's not like most transactions are confirmed within a block.

Comment (bauerj) [inline basics.rst]: You probably wanted to say 'every 10 minutes'.

File: basics.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+.. code-block:: none
+
+   #DO NOT SHARE!
+   tango execute day arena female win object expect final tank kingdom web
+
+
+Address
+-------
+
+Each seed / wallet will generate new addresses for each new transaction.
```
Comment (bauerj) [inline basics.rst]: Some people might ask whether addresses can be used multiple times. You could rephrase that so to suggest to use a different address for each transaction (for privacy reasons).

File: basics.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
+Basics
```
Comment (bauerj) [inline basics.rst]: Maybe 'glossary' would be a better title.

Comment (cluelessperson) [inline basics.rst]: agreed

File: basics.rst (hunk 2 above)
Comment (cluelessperson) [inline basics.rst]: good point

**Comment by cluelessperson** [comment]
@bauerj

I implemented all three of your suggestions, and I added some further "prettifying", in the form of horizontal lines, more graphics, and slight verbiage changes in several places.

**Pull request closed without merging**